helm_deploy/whereabouts-api/values.yaml

```diff
@@ -59,7 +59,8 @@ generic-service:
       HMPPS_SQS_QUEUES_DOMAINEVENT_QUEUE_NAME: "sqs_name"
     whereabouts-api-domain-events-sqs-dl-instance-output:
       HMPPS_SQS_QUEUES_DOMAINEVENT_DLQ_NAME: "sqs_name"
-
+    hmpps-domain-events-topic:
+      HMPPS_SQS_TOPICS_DOMAINEVENTS_ARN: "topic_arn"
 
```

Comment:
Should this be HMPPS_SNS_TOPIC_DOMAINEVENTS_ARN?  i.e. SNS, not SQS?

Reply:
And topic singular?

Reply:
Hmm interesting, will take another look, I took this from another project.

Reply:
SQS is correct.

Reply:
But SQS Simple Queue Service is about queues, and SNS is about publishing/recieving from topics - Simple Notification Service. But... I suppose it depends how the SQS library has been setup for its env vars.